src/rust/engine/process_execution/src/remote.rs

```diff
@@ -517,6 +527,8 @@ mod tests {
     address: &str,
     request: ExecuteProcessRequest,
   ) -> Result<ExecuteProcessResult, String> {
-    CommandRunner::new(address, 1).run_command_remote(request)
+    CommandRunner::new(address, 1)
+      .run_command_remote(request)
+      .wait()
```

Comment:
This should probably just return the Future so that the caller can `wait`?

Reply:
It's a test helper; I'm not sure what the benefit of having all 8 callers call `.wait()` is, when that's what they're guaranteed to do... Leaving for now, but can follow-up if you feel strongly.